client/my-sites/site-indicator/index.jsx

```diff
@@ -251,15 +251,16 @@ class SiteIndicator extends Component {
 		if ( site ) {
 			accessFailedMessage = (
 				<span>
-					{ translate( 'This site cannot be accessed.' ) }
+					{ translate( 'Jetpack is disconnected.' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **16** times:
`translate( 'Jetpack is disconnected' )` *ES Score: 10*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/jp-site-disconnected-warning)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).